docs/user-guide/test-zen-doc_Sean.md

```diff
@@ -53,50 +52,57 @@ FTP Port  |The FTP Port number for internal use. The default port is 21. If not
-Upload Zowe PAX for offline install | Upload a local (already downloaded) Zowe .PAX file using this option.               1. Click **UPLOAD PAX**. 2. Select the downloaded PAX file and click **Open**.
-SMP/E | Use this option to install Zowe through SMP/E build outside the wizard. 1. Provide the location of the Runtime Directory. 2. Click **VALIDATE LOCATION**.                          **Note** In the case of SMP/E installation, in the Initialization window, under the **Installation** tab, confirm the dataset names used during installation and click **Save**. 
+Download Zowe convenience build PAX from internet | Download the latest Zowe convenience build in .PAX format from [zowe.org](https://zowe.org) using the wizard without visiting the website.  <br /> &nbsp;&nbsp;       1 Click **License Agreement**. <br /> &nbsp;&nbsp; 2 On the End User License Agreement for Zowe page, click **AGREE**.
+Upload Zowe PAX for offline install | Upload a local (already downloaded) Zowe .PAX file using this option. <br /> &nbsp;&nbsp; 1. Click **UPLOAD PAX**.  <br /> &nbsp;&nbsp; 2. Select the downloaded PAX file and click **Open**.
+SMP/E | Use this option to install Zowe through SMP/E build outside the wizard. <br /> &nbsp;&nbsp; 1. Provide the location of the Runtime Directory. <br /> &nbsp;&nbsp; 2. Click **VALIDATE LOCATION**.                           <br /> &nbsp;&nbsp; **Note:** In the case of SMP/E installation, in the Initialization window, under the **Installation** tab, confirm the dataset names used during installation. <br /> &nbsp;&nbsp; 3. Click **Save**. 
 
 
-12. Click **Continue to Component Installation**.
+2. Click **Continue to Component Installation**.
 
 ## Zowe Server Configuration
 
-Each page of the wizard from this point onward will prompt you for configuration values for a particular step of Zowe server configuration.
-Some pages may require an additional administrator with sufficient privileges to complete the step. If you encounter such a page, you can **Skip** the page to allow the step to be completed externally to the wizard. It is also possible to input the correct values on the page, and use the **View Job** button to collect the job content to provide to an administrator without submitting it.
-Each value entered in a page will be validated, and when you complete entry on the page, the following actions can be performed:
+Perform Zowe server configuration in the wizard by providing inputs to the prompts for configuration values.
+Some steps may require an administrator with sufficient privileges to complete the step. You have the option to **Skip** the page to allow the step to be completed externally to the wizard. It is also possible to input the correct values on the page, and click **View Job** to collect the job content to provide to an administrator without submitting it.
```

Comment:
"View Job" may be missing.

Reply:
Is this "View job" option (button) missing in the UI?

Reply:
Yes @janan07 View Job feature is unavailable. all references should be gone for MVP. **View Job Output** is a feature, missing from Doc, and it is used to "view the submitted job output of an action in the wizard".